### PR TITLE
severely reduces lag on some uis

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
severely reduces lag on some uis

## Why It's Good For The Game
lag bad holy shit this gets up to 40 seconds overtime

## Testing
did it in game
## Changelog
:cl:
fix: Severely reduced lag on the seed extractor, genemodder, and biochip pad
/:cl:

### CODE REVIEW

File: code/__DEFINES/_hydroponics.dm [hunk 1]
```diff
@@ -0,0 +1,2 @@
+/// Stores a list of the cached seed icons. Used in the seed extractor and the gene modder
+GLOBAL_LIST_EMPTY(seeds_cached_base64_icons)
```
Comment (AffectedArc07) [inline code/__DEFINES/_hydroponics.dm:2]: Globals folder not defines folder

Comment (GDNgit) [reply code/__DEFINES/_hydroponics.dm:2]: 1 sec

Comment (GDNgit) [reply code/__DEFINES/_hydroponics.dm]: moved to the global list misc file

File: code/modules/hydroponics/seed_extractor.dm [hunk 2]
```diff
@@ -42,7 +42,6 @@
 
 	return FALSE
```
Comment (GDNgit) [inline code/modules/hydroponics/seed_extractor.dm:44]:
```suggestion


```

**Review by S34NW — APPROVED** [review]
Eventually want to use a modified version of bicon for this, but happy to do this for now

**Pull request merged**